scripts/build-quizes-local.ts

```diff
@@ -0,0 +1,27 @@
+import path from 'path'
+import { argv } from 'process'
+import fs from 'fs-extra'
+import { formatToCode } from './actions/utils/formatToCode'
+import { loadQuizes, resolveInfo } from './loader'
+import { supportedLocales, defaultLocale } from './locales'
+import { getQuestionFullName } from './actions/issue-pr'
+import { QuizMetaInfo } from './types'
+
+async function buildQuizesLocal() {
+  const quizesPath = path.join(__dirname, '/../quizes')
+  const locale = supportedLocales.find(locale => locale === argv[2]) ?? defaultLocale
+
+  await fs.remove(quizesPath)
+  await fs.ensureDir(quizesPath)
+
+  const quizes = await loadQuizes()
+
+  for (const quiz of quizes) {
+    const { difficulty, title } = resolveInfo(quiz, locale) as QuizMetaInfo & { difficulty: string }
```

Comment:
I cast as `QuizMetaInfo & { difficulty: string }` because `QuizMetaInfo` does not contain `difficulty`, although it probably should since `resolveInfo` does return it here. Without a lot of context, I preferred to cast locally here instead of changing the actual type based off this one case.

Reply:
I guess we could introduce a `ResolvedQuizMetaInfo` type that contains it.